### PR TITLE
ENH: output, specify samples, file prefixes

### PR DESCRIPTION
**Pull request by wasade**

### CODE REVIEW

File: scripts/mod2_pcoa.py [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
 import pandas as pd
 import seaborn as sns
+from skbio import read
 from skbio.stats.ordination import OrdinationResults
 from collections import defaultdict
 from collections import OrderedDict
 
+
 ALPHA = 1.0
 LINE_WIDTH = 0.3
```
Comment (cuttlefishh) [inline scripts/mod2_pcoa.py]: Suggest to add two lines here to define widths for dot border: 
```
LINE_WIDTH_WHITE = 2.0
LINE_WIDTH_BLACK = 1.0
```

**Comment by wasade** [comment]
@cuttlefishh, should be good to go. BTW, you have merge now

**Comment by cuttlefishh** [comment]
Awesome. I saw that. Do I need to do anything? If you send them, I can check some reports when they start coming out.

**Comment by wasade** [comment]
I've got small tweaks with the oral/skin template but it's mostly
integration stuff. Window of time this morning has diminished though

**Comment by wasade** [comment]
This is redone in #160

**Pull request closed without merging**